homeassistant/components/zwave_js/strings.json

```diff
@@ -157,7 +157,7 @@
         "step": {
           "confirm": {
             "title": "Dry and Fan preset modes will be removed: {entity_id}",
-            "description": "You are using the Dry or Fan preset modes in your entity `{entity_id}`.\n\nDry and Fan preset modes are deprecated and will be removed in a future version of Home Assistant. Please update your automations to use the corresponding Dry and Fan **HVAC modes** instead.\n\nClick on SUBMIT below once you have manually fixed this issue."
+            "description": "You are using the Dry or Fan preset modes in your entity `{entity_id}`.\n\nDry and Fan preset modes are deprecated and will be removed in Home Assistant 2024.2. Please update your automations to use the corresponding Dry and Fan **HVAC modes** instead.\n\nClick on SUBMIT below once you have manually fixed this issue."
```

Comment:
Doesn't it already show the version that will break since we set `breaks_in_ha_version`?

Reply:
Yes, we should not add it to the message.

Reply:
@MartinHjelmare @frenck - where does the HA version configured in `breaks_in_ha_version` show in the UI?  I can't see in the repair issue (see screenshot below). How can the users know how much time they have to fix the issue until the breaking change happens?

![image](https://github.com/home-assistant/core/assets/3786750/14b33992-970c-493d-8697-9d8f870481b3)

Reply:
It should show at the top of the dialog. Have you regenerated the translations with the translations script in develop mode, after the changes?

Reply:
@MartinHjelmare - yes, they were regenerated with the `python3 -m script.translations develop --all` command. Am I missing something?

Reply:
I have removed the breaking change version from the translations based on your advice, but I haven't seen the breaking change version in the dialog as you all are suggesting.